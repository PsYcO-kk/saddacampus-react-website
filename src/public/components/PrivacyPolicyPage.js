import React from 'react';

const PrivacyPolicyPage = () => (
	<div>
		<h1>Privacy Policy</h1>
	</div>
);

export default PrivacyPolicyPage;
