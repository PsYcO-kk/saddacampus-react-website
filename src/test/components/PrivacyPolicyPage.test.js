import {shallow} from 'enzyme';
import React from 'react';
import PrivacyPolicyPage from './../../components/PrivacyPolicypage.js'

test("testing for Privacy policy page component" , ()=>{
    const wrapper = shallow(<PrivacyPolicyPage />)
    expect(wrapper).toMatchSnapshot();
})