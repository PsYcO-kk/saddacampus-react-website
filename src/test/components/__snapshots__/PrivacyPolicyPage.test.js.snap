// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`testing for Privacy policy page component 1`] = `<div />`;
